README.md

```diff
@@ -1,2 +1,6 @@
 # Portfolio1
```

Comment:
- [ ] The README file is the first file a visitor sees when viewing your project, let's add the missing parts to make it more professional, here's a [template](https://github.com/microverseinc/readme-template/blob/master/README.md) you can use.